README.rst

```diff
@@ -67,36 +69,73 @@ Features
 Installation
 ============
 
-PennyLane-Forest requires both PennyLane and pyQuil. It can be installed via ``pip``:
+Installation of PennyLane-Forest, as well as all required Python packages mentioned above, can be installed via ``pip``:
+::
 
-.. code-block:: bash
+   	$ python -m pip install pennylane-forest
 
-    $ python -m pip install pennylane-forest
 
+Make sure you are using the Python 3 version of pip.
 
-.. installation-end-inclusion-marker-do-not-remove
+Alternatively, you can install PennyLane-Forest from the source code by navigating to the top directory and running
+::
+
+	$ python setup.py install
+
+Dependencies
+~~~~~~~~~~~~
+
+.. highlight:: bash
+
+PennyLane-Forest requires the following libraries be installed:
+
+* `Python <http://python.org/>`_ >=3.6
+
+as well as the following Python packages:
+
+* `PennyLane <http://pennylane.readthedocs.io/>`_
+* `pyQuil <http://docs.rigetti.com/>`_
+
+If you currently do not have Python 3 installed, we recommend
+`Anaconda for Python 3 <https://www.anaconda.com/download/>`_, a distributed version
+of Python packaged for scientific computation.
+
+Additionally, if you would like to compile the quantum instruction language (Quil) and run it
+locally using a quantum virtual machine (QVM) server, you will need to download and install the
+Forest software development kit (SDK):
 
-Getting started
-===============
+* `Forest SDK <https://pyquil-docs.rigetti.com/en/stable/>`_
 
-Once the PennyLane-Forest plugin is installed, the three provided pyQuil devices can be accessed straight away in PennyLane.
+Alternatively, you may sign up for Rigetti's Quantum Cloud Services (QCS) to acquire a Quantum Machine
+Image (QMI) which will allow you to compile your quantum code and run on real quantum processing units (QPUs),
+or on a preinstalled QVM. Note that this requires a valid QCS account.
 
-You can instantiate these devices for PennyLane as follows:
+* `Quantum Cloud Services <https://www.rigetti.com/>`_
```

Comment:
Here, perhaps the Rigetti docs could be nice to link to. Since the first heading is "Quantum Cloud Services Documentation".
```suggestion
* `Quantum Cloud Services <https://docs.rigetti.com/en/>`_
```